### DISCUSSION TITLE
Use of flood inside Docker

### DISCUSSION THREAD

**Discussion by romancin** [jesec/flood#76] [Q&A] [answered]
Type: Question

## Question

Hi! I have just migrated my container to your frok of Flood. Thank you very much for your work!

I have doubts configuring flood with your version. Until changed to this repo, I was using a config.js file that I configured just once, and when delete the container, the configuration stored in this file is loaded again, so it is persistant to container recreations.

I have been reading the config.cli.js file, but it is not clear to me how to apply the old settings in this new way. 

Could you please help me?

Thank you very much in advance!

**Comment by jesec** [comment, marked as answer]
Check out https://github.com/jesec/flood/blob/master/config.d.ts for more information about config.

CLI argument or environment variable is the recommended way but if necessary you can still use the static configuration.

**Comment by jesec** [comment]
Additionally, Flood no longer depends on build-time configurations. As such, you can install Flood easily with `sudo npm i -g flood` (or `@jesec/flood` for cutting-edge builds) in the Docker. Then you can simply execute `flood` to launch Flood. `flood` accepts CLI arguments and environment variables for configuration.
